guides/vpn/README.md

```diff
@@ -37,39 +37,60 @@ That is it! You should now be connected to the `Made Tech - VPN`. If you are exp
 
 ### Ubuntu/Debian
 
-https://docs.foxpass.com/docs/foxpass-linux-l2tpipsec
```

Comment:
If we are committing to maintaining our own copy of the ubuntu guide then can we keep the link in case it falls out of date with the official Foxpass one? The VPN backend uses Foxpass so they may change their settings in a future update.